x/superfluid/types/msg_test.go

```diff
@@ -73,3 +75,70 @@ func TestAuthzMsg(t *testing.T) {
 		})
 	}
 }
+
+func TestUnbondConvertAndStakeMsg(t *testing.T) {
```

Comment:
lets test that blank val address works, so it would have caught the bug introduced earlier.

Reply:
Added in https://github.com/osmosis-labs/osmosis/pull/5949/commits/18f4bd7e81d8dd238fc541fa4d9cd840ca82d0ed !